stx/ceph-rest-api

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+
+### BEGIN INIT INFO
+# Provides:          ceph-rest-api
+# Required-Start:    $ceph
+# Required-Stop:     $ceph
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Ceph REST API daemon
+# Description:       Ceph REST API daemon
+### END INIT INFO
+
+DESC="ceph-rest-api"
+DAEMON="/usr/bin/ceph-rest-api"
+RUNDIR="/var/run/ceph"
+PIDFILE="${RUNDIR}/ceph-rest-api.pid"
```

Comment:
this ceps-rest-api.pid path is NOT consistent with the path you specified in "service-mgmt/sm-db-1.0.0/database/create_sm_db.sql"
see https://review.openstack.org/#/c/634120/ in details.
Either way you need to modify one place or another.

Reply:
@yhu6
1) Currently, the system won't execute ceph-rest-api even ceph-rest-api file are built into system 
2) I'm going to delete stx/{ceph-rest-api,ceph-rest-api.service} file from source code.
3) stx/mgr-restful-plugin has set the PID file as /var/run/ceph/mgr/mgr-restful-plugin.pid

Have you hit error in the running system?

Reply:
just check this path (/var/run/ceph/mgr/mgr-restful-plugin.pid) to make sure it is the same path in sm db file.